tool/plugin/lib/plugin.dart

```diff
@@ -383,7 +464,7 @@ class BuildCommand extends ProductCommand {
       log('spec.version: ${spec.version}');
 
       var compileFn = () async {
-        var r = await runner.javac2(spec);
+        var r = await externalBuildCommand(spec);
```

Comment:
not your fault, but can these both be final?

Reply:
Done